### ISSUE TITLE
Importing vlan resource without a name crashes terraform

### ISSUE THREAD

**Issue by chanster** [IBM-Cloud/terraform-provider-ibm#1067]
Importing a `ibm_network_vlan` that does not have a `name` set fails.

Example of `ibmcloud sl vlan list`
```
ID        number   name           firewall   network_space   primary_router   hardware   virtual_servers   public_ips
2567033   3224     my-vlan1       No         PRIVATE         [REMOVED]        0          2                 61
2737798   3238     -              No         PRIVATE         [REMOVED]        0          4                 61
```

### Terraform Version
Terraform v0.12.20
+ provider.ibm v1.1.0

### Affected Resource(s)
- ibm_network_vlan

### Terraform Configuration Files
```hcl
provider "ibm" {
  version = "~> 1.1"
}

data "ibm_network_vlan" "vlan_2737798" {
  id = 2737798
}

resource "ibm_network_vlan" "vlan_2737798" {
  id = data.ibm_network_vlan.vlan_2737798.id
}
```

### Panic Output
https://gist.github.com/chanster/9401fd446ff7ef03ec78ada16dad1cad

### Expected Behavior
Import of `ibm_network_vlan` should complete

### Actual Behavior
Import fails with crash error.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create a IBM Cloud Classic Infrastructure VLAN without a name.
1. Create data source in Terraform config
1. Create resource in Terraform config
1. run `terraform import ibm_network_vlan.[resource] [vlan_id]`

### Important Factoids
These VLANs were created manually through the [IBM Cloud](https://cloud.ibm.com) web interface.

Work around is to add a name to the VLAN after it has been created.

**Comment by hkantare** [comment]
We already have a fix in master branch 
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/1054

**Comment by hkantare** [comment]
It will be available in next release

**Comment by chanster** [comment]
Fixed in v1.2.0
